src/pages/imageSetup/components/layout/FunnelLayout.tsx

```
@@ -1,16 +1,37 @@
+import { overlay } from 'overlay-kit';
```

ESLint

[error] Replace `'overlay-kit'` with `"overlay-kit"`

(prettier/prettier)